### PR TITLE
[bitnami/kubeapps] Update the cert-manager annotations

### PR DESCRIPTION
**Pull request by dud225**
**Description of the change**

According to the documentation:
https://cert-manager.io/docs/usage/ingress/#optional-configuration,
the `kubernetes.io/tls-acme` annotations pertains to the kube-lego
project https://github.com/jetstack/kube-lego which is no longer maintained
and has been displaced by cert-manager.
As a result the cert-manager annotations should be used instead.

### CODE REVIEW

**Comment by antgamdia** [comment]
Additional question: are we planning to move from the [deprecated annotation](https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation) `kubernetes.io/ingress.class` to the `ingressClassName` prop?

File: bitnami/kubeapps/templates/ingress.yaml [hunk 1]
```diff
@@ -9,8 +9,16 @@ metadata:
...
+      {{- with .clusterIssuer }}
+    cert-manager.io/cluster-issuer: {{ . }}
+      {{- end }}
+      {{- with .httpIngressClass }}
+    acme.cert-manager.io/http01-ingress-class: {{ . }}
+      {{- end }}
+      {{- with .commonName }}
+    cert-manager.io/common-name: {{ . }}
+      {{- end }}
     {{- end }}
```
Comment (migruiz4) [inline bitnami/kubeapps/templates/ingress.yaml:22]: To keep the same style as the other charts, could you please use the following syntax instead?
```suggestion
    {{- if .Values.ingress.certManager.clusterIssuer }}
    cert-manager.io/cluster-issuer: {{ .Values.ingress.certManager.clusterIssuer }}
    {{- end }}
    {{- if .httpIngressClass }}
    acme.cert-manager.io/http01-ingress-class: {{ .Values.ingress.certManager.httpIngressClass }}
    {{- end }}
    {{- if .Values.ingress.certManager.commonName }}
    cert-manager.io/common-name: {{ .Values.ingress.certManager.commonName }}
    {{- end }}
```

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Hi @dud225,

Thank you very much for your contribution! 

I left a suggestion below regarding style.
Additionally, this PR is missing a minor version bump in `bitnami/kubeapps/Chart.yaml`.

**Comment by migruiz4** [comment]
@antgamdia regarding your question, we are currently working on a round of 'Charts standardizations' which would include the addition of `ingressClassName` for all the charts among other features.

File: bitnami/kubeapps/templates/ingress.yaml (hunk 1 above)
Comment (dud225) [reply bitnami/kubeapps/templates/ingress.yaml]: Hello
Done. I had to leave a top-level `{{- if .Values.ingress.certManager }}` condition as this value may missing.

Comment (dud225) [reply bitnami/kubeapps/templates/ingress.yaml]: Nevermind my last comment, I've noticed that the CI failed because of that, so I've set a default value.

**Comment by migruiz4** [comment]
Hi @dud225,

Thank you very much for applying the suggested changes! Your contribution looks good to me.

I'd like to request a review from @alvneiayu, as we may need to apply similar changes to how we configure cert-manager in other charts.

File: bitnami/kubeapps/Chart.yaml [hunk 2]
```diff
@@ -33,4 +33,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 7.4.0
+version: 7.4.1
```
Comment (migruiz4) [inline bitnami/kubeapps/Chart.yaml:36]: Bump a minor version please (`7.5.0`)

**Review by migruiz4 — COMMENTED** [review]
Just a small change before approving.

**Review by migruiz4 — APPROVED** [review]

**Review by alvneiayu — APPROVED** [review]
LGTM

**Pull request merged**